use_case_examples/titanic/Makefile

```diff
@@ -2,11 +2,12 @@
 export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 
-EXAMPLE_NAME=cifar_brevitas_finetuning
```

Comment:
Why you removed `cifar_brevitas_finetuning` ?

Reply:
This is in the titanic usecase. It was a bad copy paste I suppose.

Reply:
Oh yes, I saw it many times, and didn't pay attention to the file name.

Good catch !